lib/shared/errors.js

```diff
@@ -228,7 +234,7 @@ exports.getDescription = (error) => {
   }
 
   if (error.stack) {
```

Comment:
Not quite. Note we explicitly return an empty string if there is a stack but we're not considering it.